proto/inference_service_pb2_grpc.py

```diff
@@ -0,0 +1,84 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
should these generated files be in the .gitignore?  Should they be checked in?  I don't know :)